src/ua.c

```diff
@@ -126,7 +126,8 @@ void ua_printf(const struct ua *ua, const char *fmt, ...)
 		return;
 
 	va_start(ap, fmt);
-	info("%r@%r: %v", &ua->acc->luri.user, &ua->acc->luri.host, fmt, &ap);
+	info("%p %r@%r: %v", ua, &ua->acc->luri.user, &ua->acc->luri.host, fmt,
+			&ap);
```

Comment:
I think that printing a pointer to the console (read by an enduser) is a bit user-unfriendly ...

we should find a better way to identify each user-agent, for example index

Reply:
Yes, I agree! But then I didn't understand what was your intention with
```
struct ua *ua = carg->data;
```
How should the pointer carg->data be set?
Or, should we replace this and read an index from carg->prm? And in ctrl_tcp/ctrl_dbus set the index to carg->prm?